webpack.config.js

```diff
@@ -139,18 +148,24 @@ module.exports = function makeWebpackConfig () {
    * Reference: https://github.com/postcss/autoprefixer-core
    * Add vendor prefixes to your css
    */
-  config.postcss = [
-    autoprefixer({
-      browsers: ['last 2 version']
-    })
-  ];
+   // NOTE: This is now handled in the `postcss.config.js`
+   //       webpack2 has some issues, making the config file necessary
```

Comment:
I will find the reasoning behind this and send it to you.   I applied this fix personally sometime ago (when working with webpack beta release) and have not revistited